### ISSUE TITLE
Weird vertical scrollbar shows up in summary chart ... sometimes

### ISSUE THREAD

**Issue by jasonrhodes** [SparkPost/2web2ui#54]
<img width="938" alt="screen shot 2017-08-26 at 5 26 59 pm" src="https://user-images.githubusercontent.com/159370/29745253-de127854-8a83-11e7-9ded-25d2b51018a8.png">

Looks like this is the problem: https://github.com/maslianok/react-resize-detector/issues/6

Apparently their "solution" was to add this:
```css
.recharts-responsive-container div {
  overflow: hidden !important;
}
```

:sob:

**Comment by jasonrhodes** [comment]
UPDATE: react-resize-detector released version 0.6.0 which fixes this bug! I submitted a PR to recharts to upgrade to the new version.

https://github.com/recharts/recharts/pull/891

🤞

**Comment by jonambas** [comment]
Looks like the PR was merged
